src/app/examples/crud/components/crud.component.html

```diff
@@ -0,0 +1,80 @@
+        <h2>{{ 'anms.examples.crud.subtitle2' | translate }} <span *ngIf="selectedBook"> : {{selectedBook.id}} </span></h2>
+        <span class="d-flex justify-content-end">
+          <span>
+            <button mat-icon-button color="accent" (click)="isEditing=!isEditing">
+              <mat-icon fontSet="fas" fontIcon="fa-eye" *ngIf="isEditing"></mat-icon>
+              <mat-icon fontSet="fas" fontIcon="fa-edit" *ngIf="!isEditing"></mat-icon>
+            </button>
+          </span>
+          <button mat-icon-button color="warn" (click)=delete(bookForm) *ngIf="selectedBook">
+            <mat-icon fontSet="fas" fontIcon="fa-trash"></mat-icon>
+          </button>
+        </span>
+      </span>
+      <form #bookForm="ngForm" [formGroup]="bookFormGroup" (ngSubmit)="onSubmit()" [style.display]="isEditing ? 'inherit' : 'none'">
+        <div class="row">
+          <mat-form-field class="col" [ngClass]="routeAnimationsElements">
+            <input matInput placeholder="{{ 'anms.examples.crud.placeholder1' | translate }}" formControlName="title"
+              required>
+            <mat-error *ngIf="bookFormGroup.get('title').invalid">{{ 'anms.examples.crud.placeholder1' | translate }}
+              {{ 'anms.examples.crud.error' | translate }}</mat-error>
+          </mat-form-field>
+        </div>
+        <div class="row">
+          <mat-form-field class="col" [ngClass]="routeAnimationsElements">
+            <input matInput placeholder="{{ 'anms.examples.crud.placeholder2' | translate }}" formControlName="author"
+              required>
+            <mat-error *ngIf="bookFormGroup.get('author').hasError('required')">{{ 'anms.examples.crud.placeholder2' |
+              translate }}
+              {{ 'anms.examples.crud.error' | translate }}</mat-error>
+          </mat-form-field>
+        </div>
+        <div class="row">
+          <mat-form-field class="col" [ngClass]="routeAnimationsElements">
+            <textarea matInput placeholder="{{ 'anms.examples.crud.placeholder3' | translate }}" formControlName="description"
+              rows="5"></textarea>
+            <mat-error *ngIf="bookFormGroup.get('description').invalid">{{ 'anms.examples.crud.placeholder1' |
+              translate }}
+              {{ 'anms.examples.crud.error' | translate }}</mat-error>
+          </mat-form-field>
+        </div>
+        <div class="row d-flex justify-content-around">
+          <button type="button" mat-raised-button [ngClass]="routeAnimationsElements" (click)="clearForm(bookForm)"
+            color="accent">{{'anms.examples.crud.new' | translate }}
+          </button>
+          <button mat-raised-button color="primary" [ngClass]="routeAnimationsElements">{{
+            'anms.examples.crud.save' | translate }}
+          </button>
+        </div>
+      </form>
+      <div [ngClass]="routeAnimationsElements" [style.display]="isEditing ? 'none' : 'inherit'">
+        <h3>{{selectedBook?.title}}</h3>
```

Comment:
Perhaps it would be better to use an `*ngIf` on the div? Instead of the `?.` operator